packages/@adobe/spectrum-css-temp/components/card/index.css

```diff
@@ -1241,3 +1241,26 @@ For now working around it with a useLayoutEffect in CardBase.
   }
 }
 /** END Standard Horizontal **/
+
+/* CardView styles */
+.spectrum-CardView {
+  outline: none;
+
+  .spectrum-CardView-centeredWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+  }
+
+  .spectrum-CardView-CardWrapper {
+    overflow: visible !important;
```

Comment:
it was more about the important, haha *cringe*

Reply:
ah, yeah unfortunately I had to use important because the overflow + contain styles that interfere with this are inline styles provided to the div via VirtualizerItem.